client/src/styles/Home.css

```diff
@@ -32,3 +32,8 @@
   text-align: center;
   margin-top: 20px;
 }
+
+.footerStyle {
+  text-decoration: underline;
+  color: white;
```

Comment:
Can you make sure that when we mouse hover the link, the color stay white but the text is not underline? Currently the link is blue.

Reply:
iI changed it